src/overlays/actors/ovl_Bg_Gnd_Darkmeiro/z_bg_gnd_darkmeiro.c

```diff
@@ -35,25 +35,225 @@ const ActorInit Bg_Gnd_Darkmeiro_InitVars = {
     (ActorFunc)BgGndDarkmeiro_Update,
     NULL,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Gnd_Darkmeiro/func_80878C30.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Gnd_Darkmeiro/BgGndDarkmeiro_Init.s")
+extern UNK_TYPE D_0600C080;
+extern UNK_TYPE D_060088B0;
+extern UNK_TYPE D_0600BEC0;
+
+void BgGndDarkmeiro_TogglePlatform(BgGndDarkmeiro *this, GlobalContext *globalCtx) {
```

Comment:
We use the `type* name` syntax for pointers. Make sure to run `format.sh` before making a PR so your files are formatted accordingly to our style conventions.
```suggestion
void BgGndDarkmeiro_TogglePlatform(BgGndDarkmeiro* this, GlobalContext* globalCtx) {
```